src/types/button.ts

```diff
@@ -11,3 +16,13 @@ export enum ButtonSize {
   LARGE = 'large',
   MINI = 'mini',
 }
+
+export interface ButtonProps {
+  disabled?: boolean;
+  endEnhancer?: React.ReactElement | string;
+  isLoading?: boolean;
+  kind?: ButtonKind;
+  onClick?: (event: GestureResponderEvent) => void;
+  size?: ButtonSize;
+  startEnhancer?: React.ReactElement | string;
```

Comment:
Do we need `string`? I believe `React.ReactElement` (or `ReactNode`) should be sufficient?